File: Add/add.ts
function add(para1 : number, para2 : number) : number{
    return para1 + para2;
}

function addAny(...para1 : number[]) : number{
    let total = 0;

    para1.forEach((num) => {
        total += num;
    })

    return total;
}

console.log(add(2,3));
console.log(addAny(2,3,5,6,7,8));

File: Add/add.java
// package Add;

public class add{

    public static int addValue(int ... a){
        int total = 0;
        for(int num : a){
            total += num;
        }
        return total;
    }

    public static int addValue(int a, int b){
        return a+b;
    }

    public static void main(String[] args) {
        int a = addValue(2,3,45,6,7);
        System.out.println(a);
    }
}